boards/Kconfig

```diff
@@ -99,6 +99,11 @@ config QEMU_UEFI_BOOT
 	  This option indicates that QEMU will use UEFI bootable method
 	  to boot up.
 
+config QEMU_KVM
+	bool "QEMU uses KVM acceleration"
+	help
+	  This option enables KVM acceleration in QEMU
```

Comment:
Since this feature only works for ARM64, it should depend on ARM64.

Also looking at the implementation in this PR, I am starting to question the usefulness of this Kconfig since this needs to be handled on per-board basis anyway (i.e. you simply cannot take `qemu_cortex_a53` board and make it KVM, that does not make sense).